src/Serializer/SerializerContextBuilder.php

```diff
@@ -49,6 +49,10 @@ public function createFromRequest(Request $request, bool $normalization, array $
             $context['item_operation_name'] = $attributes['item_operation_name'];
         }
 
+        if (!$normalization && (!isset($context['allow_update']) || !is_bool($context['allow_update']))) {
```

Comment:
IMO it's not necessary to test if `allow_update` is a boolean, and it's weird to silently override it. I would remove this test.